### ISSUE TITLE
Set application_name for Postgres connection

### ISSUE THREAD

**Issue by mbanck** [wrouesnel/postgres_exporter#352]
It would be nice if postgres_exporter would set an application (like `postgres-exporter` or so), so that it can be better identified in `pg_stat_activity` amonst other backends.

**Comment by neel5481** [comment]
AFAIK - you can always set "PGAPPNAME" environment variable before starting pg exporter. That configured name will be displayed in "pg_stat_activity".

**Comment by cosenmarco** [comment]
Suggestion works fine. Would be nice to have it documented on the README
